mavros_gym:
  px4-est: 'ekf2'
  use_pose_estimator: True
  environment_name: 'uav-follow-trajectory-env-v0'
  running_step: 0.04 # amount of time the control will be executed
  pos_step: 0.016     # increment in position for each command
  
  #qlearn parameters
  alpha: 0.1
  gamma: 0.7
  epsilon: 0.9
  epsilon_discount: 0.999
  n_eps: 500
  max_episode_steps: 1000
  number_splits: 10 #set to change the number of state splits for the continuous problem and also the number of env_variable splits

  geodesic_distance: False

  #Action Space Ranges
  lxy_vel_range: 20
  lz_vel_range: 6
  rot_vel_range: 6

  linear_forward_speed: 0.5 # Spwwed for ging fowards
  angular_turn_speed: 0.05 # Lienare speed when turning
  angular_speed: 0.3 # Angular speed when turning Left or Right

  init_speed_vector:
    linear_x: 0.0
    linear_y: 0.0
    linear_z: 0.0
    angular_x: 0.0
    angular_y: 0.0
    angular_z: 0.0
  
  work_space: # 3D cube in which Drone is allowed to move
    x_max: 13.0
    x_min: -1.0
    y_max: 5.0
    y_min: -1.0
    z_max: 3.0
    z_min: -1.0

  max_orientation_w: 99999999
  max_orientation_x: 99999999
  max_orientation_y: 99999999
  max_orientation_z: 99999999

  max_roll: 1.57 # Max roll after which we end the episode
  max_pitch: 1.57 # Max roll after which we end the episode
  max_yaw: inf # Max yaw, its 4 because its bigger the pi, its a complete turn actually the maximum

  desired_position:
    x: 10.0
    y: 5.0
    z: 9.0
  desired_orientation:
    w: 1.0
    x: 0.0
    y: 0.0
    z: 0.0

  desired_point_epsilon: 0.05 # Error acceptable to consider that it has reached the desired point

  closer_to_point_reward: 10 # We give points for getting closer to the desired point
  not_ending_point_reward: 1 # Points given if we just dont crash
  end_episode_points: 200 # Points given when ending an episode


